e2e/core/ui/CommonActions.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+//TODO set uppercase const
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)